pinot-common/pom.xml

```diff
@@ -312,6 +312,10 @@
                       <pattern>com.google.common.base</pattern>
                       <shadedPattern>shaded.com.google.common.base</shadedPattern>
                     </relocation>
+                    <relocation>
```

Comment:
I think you also need to add this into the `pinot-plugins/pom.xml`

Reply:
I just want to fix the current error I faced and limit the side effects. I'll keep that in mind if we see another issue with a different module. I was able to get the local test working with only `pinot-common` 